src/components/LoginModal/LoginModal.component.jsx

```diff
@@ -7,18 +7,20 @@ import Button from '@components/Button';
 
 function LoginModal(props) {
   const { login } = useAuth();
-  const [username, setUsername] = useState('');
-  const [password, setPassword] = useState('');
+  const [username, setUsername] = useState('wizeline');
+  const [password, setPassword] = useState('Rocks!');
```

Comment:
Did we move test credentials as default values to log in faster by clicking the sign-in button?

Reply:
Yep due this project is more an exercise, it is just for faster login.